package.json

```diff
@@ -59,6 +59,7 @@
     "eventemitter3": "5.0.1",
     "expo": "53.0.22",
     "expo-application": "6.1.5",
+    "expo-asset": "11.1.7",
```

Comment:
I had to add this because several of the tests required expo font, and expo font required this...im not sure why these tests were failing in the pr...